src/Microsoft.AspNetCore.Sockets.Common/Internal/Formatters/ServerSentEventsMessageFormatter.cs

```diff
@@ -14,11 +14,11 @@ public static class ServerSentEventsMessageFormatter
         private static readonly byte[] DataPrefix = new byte[] { (byte)'d', (byte)'a', (byte)'t', (byte)'a', (byte)':', (byte)' ' };
         private static readonly byte[] Newline = new byte[] { (byte)'\r', (byte)'\n' };
 
-        private const byte LineFeed = (byte)'\n';
-        private const char TextTypeFlag = 'T';
-        private const char BinaryTypeFlag = 'B';
-        private const char CloseTypeFlag = 'C';
-        private const char ErrorTypeFlag = 'E';
+        static readonly byte LineFeed = (byte)'\n';
+        static readonly char TextTypeFlag = 'T';
+        static readonly char BinaryTypeFlag = 'B';
+        static readonly char CloseTypeFlag = 'C';
+        static readonly char ErrorTypeFlag = 'E';
```

Comment:
Add `private` to these. We always put accessibility/visibility modifiers on, even when they are the default.